airbyte-integrations/connectors/source-bigquery/src/main/resources/spec.json

```diff
@@ -18,12 +18,12 @@
       },
       "dataset_id": {
         "type": "string",
-        "description": "The BigQuery Dataset ID to look for tables to replicate from.",
-        "title": "Default Dataset ID"
+        "description": "The schema name if only one schema is interested. Dramatically boost source discover operation.",
+        "title": "Default Dataset ID [Optional]"
```

Comment:
I'm not sure if we have to specify that this field is `Optional`, since we indicate the required fields at the top in the ``required` field and they are highlighted on the UI.

Reply:
removed [Optional]